src/config.ts

```diff
@@ -104,3 +104,9 @@ export const providersUrls = {
 export const getUrlByKey = (key: HotelProviders): string => {
   return providersUrls[key];
 };
+
+// Rewards
+export const rewardPercentage = Number(process.env.REWARD_PERCENTAGE);
```

Comment:
need to add this url fill check here:
![image](https://user-images.githubusercontent.com/43822362/186836101-32f13acb-1377-4a6a-8b20-b243fbeb28f2.png)